sdk/tables/azure-data-tables/src/serializers.cpp

```diff
@@ -527,7 +527,21 @@ namespace Azure { namespace Data { namespace Tables { namespace _detail {
   {
     Models::TableEntity tableEntity{};
 
-    auto properties = json.get<std::map<std::string, std::string>>();
+    std::map<std::string, std::string> properties;
+    for (auto it = json.items().begin(); it != json.items().end(); ++it) 

```

Comment:
Possibly consider using iterator syntax rather than the older enumeration logic? At a minimum, I believe it will save a copy assignment on the JSON value.

```
    for (auto &it = json.items()) 
```

Reply:
on it

Reply:
If it doesn't work, then so be it. Don't stress over this nit comment.

Reply:
> If it doesn't work, then so be it. Don't stress over this nit comment.

Larry for the time being let us push this... I will add another PR once i get some free time for this..

Regards,
Arwell